src/main/java/com/parasoft/findings/jenkins/coverage/api/metrics/steps/CoverageReporter.java

```diff
@@ -63,29 +58,12 @@ public CoverageBuildAction publishAction(final String id, final String optionalN
 
             Node modifiedLinesCoverageRoot = rootNode.filterByModifiedLines();
 
-            NavigableMap<Metric, Fraction> modifiedLinesCoverageDelta;
-            List<Value> aggregatedModifiedFilesCoverage;
-            NavigableMap<Metric, Fraction> modifiedFilesCoverageDelta;
-            if (hasModifiedLinesCoverage(modifiedLinesCoverageRoot)) {
-                Node modifiedFilesCoverageRoot = rootNode.filterByModifiedFiles();
-                aggregatedModifiedFilesCoverage = modifiedFilesCoverageRoot.aggregateValues();
-                modifiedFilesCoverageDelta = modifiedFilesCoverageRoot.computeDelta(rootNode);
-                modifiedLinesCoverageDelta = modifiedLinesCoverageRoot.computeDelta(modifiedFilesCoverageRoot);
-            }
-            else {
-                modifiedLinesCoverageDelta = new TreeMap<>();
-                aggregatedModifiedFilesCoverage = new ArrayList<>();
-                modifiedFilesCoverageDelta = new TreeMap<>();
-                if (rootNode.hasModifiedLines()) {
-                    log.logInfo("No detected code changes affect the code coverage");
-                }
+            if (rootNode.hasModifiedLines()) {
```

Comment:
This `if` is only executed when there's no line coverage change compare to the reference build, so we need to keep the `hasModifiedLinesCoverage(modifiedLinesCoverageRoot)` check.